examples/platforms/posix/random.c

```diff
@@ -65,3 +66,20 @@ uint32_t otPlatRandomGet(void)
 
     return mlcg;
 }
+
+ThreadError otPlatSecureRandomGet(uint16_t aInputLength, uint8_t *aOutput, uint16_t *aOutputLength)
+{
+    ThreadError error = kThreadError_None;
+    uint16_t length = 0;
+
+    VerifyOrExit(aOutput && aOutputLength, error = kThreadError_InvalidArgs);
+
+    while (length < aInputLength)
```

Comment:
Could I know what is the reason for using for-loop here? It is only a matter of pragmatic, but not performance or other reasons, right?